rust-toolchain.toml

```diff
@@ -0,0 +1,14 @@
+[toolchain]
+channel = "nightly"
```

Comment:
Uhm… why do we have a root toolchain file now (https://github.com/paritytech/substrate/pull/13580#issuecomment-1465012700)?  
And why is it set to nightly?

Reply:
Hm. It's set to nightly because the docs said to use nightly when I authored it. I guess it's easier if I split it and restore the original scope of the PR that only modifies the template?

Reply:
> Hm. It's set to nightly because the docs said to use nightly when I authored it. I guess it's easier if I split it and restore the original scope of the PR that only modifies the template?

Yes please let's do this for now.

Reply:
done.